astore/server/auth/factory.go

```diff
@@ -79,6 +78,9 @@ func WithTimeLimit(limit time.Duration) Modifier {
 
 func WithCA(fileContent []byte) Modifier {
 	return func(server *Server) error {
+		if len(fileContent) == 0 {
+			return nil
```

Comment:
does server has a `Log`/`Logger` member? If yes, add a:
```
server.Warnf("CA file not found - will operate without certificates")
```

Reply:
It did not, now it does. Although passing it in isn't supported right now with the way the auth server is instantiated, it'll include the normal logger at a later date. Probably during spring cleaning.